bin/slurm-send-mail.py

```diff
@@ -239,26 +264,53 @@ def runCommand(cmd):
+                                                                                        jobDic[x[0]] = x[1]
+                                                                        stdoutFile = jobDic['StdOut']
+                                                                        stdoutFiles += stdoutFile
+                                                                        logging.info('stdoutFile: {}'.format(stdoutFile))
+                                                                        stderrFile = jobDic['StdErr']
+                                                                        stderrFile = '' if stderrFile == stdoutFile else stderrFile
+                                                                        stderrFiles += stderrFile
+                                                                        try:
+                                                                            with open(stdoutFile, 'r') as fread:
+                                                                                stdout_last_100 += stdoutFile+':\n'
+                                                                                for i, line in enumerate(fread):
+                                                                                    stdout_last_100 += line
+                                                                                    if i == 100:
+                                                                                        break
+                                                                            stdout_last_100 += '\n\n'
+                                                                        except:
+                                                                            logging.exception("couldn't read the stdout file")
+                                                                        if stderrFile != '':
+                                                                            try:
+                                                                                with open(stderrFile, 'r') as fread:
+                                                                                    stderr_content += stderrFile+':\n'
+                                                                                    stderr_content += fread.read()+'\n\n'
+                                                                            except:
+                                                                                logging.exception("couldn't read the stderr file")
+                                                                else:
+                                                                        logging.error('failed to run: %s' % cmd)
+                                                                        logging.error(stdout)
+                                                                        logging.error(stderr)
```

Comment:
Nice idea adding some of the standard out to the e-mail. One slight issue though is that it is not always the case that the server running `slurmctld` also mounts the file system where the user's job output has been written to. For example, on supercomputers that use Slurm, often the parallel file system (e.g. Lustre) where the user's job output is written to is not mounted on the Slurm servers. I think it would better to remove this code for now in this particular pull request and just focus on job arrays.